bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -121,6 +122,46 @@ contract("Bridge NFT", async function(accounts) {
       assert.equal(result, "v1");
     });
 
+    it("should call the token creator", async function () {
+      const testTokenCreator = await TestTokenCreator.new({ from: anAccount });
+      const tokenCreator = await this.NFTBridge.getTokenCreator(testTokenCreator.address, defaultTokenId);
+      assert.equal(tokenCreator, anAccount);
+    });
+
+    it("should change the Allow Tokens", async function() {

```

Comment:
I'd maybe rename to "should change the allow tokens as expected when calling changeAllowTokens"

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/268/commits/9c4e1c7b64b7379ff804bb3f768eed9c96a7461b